funnel/assets/js/project_header.js

```diff
@@ -1,7 +1,156 @@
-import { SaveProject, Video } from './util';
+import { Utils, SaveProject, Video } from './util';
+
+const Ticketing = {
+  init(tickets) {
+    if (tickets.boxofficeUrl) {
+      this.initBoxfficeWidget(tickets);
+    }
+
+    this.initTicketModal();
+  },
+
+  initBoxfficeWidget({
+    boxofficeUrl,
+    widgetElem,
+    org,
+    itemCollectionId,
+    itemCollectionTitle,
+  }) {
+    let url;
+
+    if (boxofficeUrl.slice(-1) === '/') {
+      url = `${boxofficeUrl}boxoffice.js`;
+    } else {
+      url = `${boxofficeUrl}/boxoffice.js`;
+    }
+
+    $.get({
+      url,
+      crossDomain: true,
+      timeout: window.Hasgeek.config.ajaxTimeout,
+      retries: 5,
+      retryInterval: window.Hasgeek.config.retryInterval,
+
+      success(data) {
+        const boxofficeScript = document.createElement('script');
+        boxofficeScript.innerHTML = data.script;
+        document.getElementsByTagName('body')[0].appendChild(boxofficeScript);
+      },
+
+      error(response) {
+        const ajaxLoad = this;
+        ajaxLoad.retries -= 1;
+        let errorMsg;
+
+        if (response.readyState === 4) {
+          errorMsg = 'Server error, please try again later.';
+          $(widgetElem).html(errorMsg);
+        } else if (response.readyState === 0) {
+          if (ajaxLoad.retries < 0) {
+            if (!navigator.onLine) {
+              errorMsg = 'Unable to connect. There is no network!';
+            } else {
+              errorMsg =
+                '<p>Unable to connect. If you are behind a firewall or using any script blocking extension (like Privacy Badger), please ensure your browser can load boxoffice.hasgeek.com, api.razorpay.com and checkout.razorpay.com .</p>';
```

Comment:
gettext, and no `<p>` tag inside the translation. Replace "If _you_ are behind" with "If _this device_ is behind"